common/djangoapps/third_party_auth/tests/test_pipeline_integration.py

```diff
@@ -5,11 +5,11 @@
 from django.conf import settings
 from django import test
 from django.contrib.auth import models
-import mock
+from mock import MagicMock, patch
```

Comment:
Not strictly a necessary change, but it makes the tests better overall, so I'm okay with it.